store/storetest/post_store.go

```diff
@@ -94,6 +94,7 @@ func testPostStoreSave(t *testing.T, ss store.Store) {
 		replyPost.Message = "zz" + model.NewId() + "b"
 		replyPost.RootId = rootPost.Id
 
+		time.Sleep(2 * time.Millisecond)
```

Comment:
Would a delay of one millisecond be enough?

Reply:
Yes, should be enough